python/turbine_models/version_info.json

```diff
@@ -0,0 +1,3 @@
+{
+    "package-version": "0.0.1.dev1"
```

Comment:
Let's use the version_info.json from the root directory. I think we should be releasing these in step with each other to avoid the whole torch/torchvision debacle.